.github/workflows/build.yaml

```
@@ -30,138 +34,299 @@ concurrency:
-          path: output/${{ env.ARTIFACT_NAME_CRANECTLD_DEB }}
-          retention-days: 14
-          if-no-files-found: error
+      # Generate meta.json (human-readable metadata; replaces fingerprint.txt)
+      - name: Generate meta.json
+        run: |
+          TS="$(date -u +"%Y-%m-%dT%H:%M:%SZ")"
+          RUN_URL="https://github.com/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
+          SIZE_BYTES=$(stat -c %s build-output.tar.gz)
+          SHA256=$(sha256sum build-output.tar.gz | awk '{print $1}')
+          cat > meta.json <<EOF
+{
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 231: could not find expected ':'

(syntax-check)